opencti-platform/opencti-graphql/src/manager/publisherManager.ts

```diff
@@ -144,9 +145,27 @@ export async function handleEmailNotification(
 ) {
   const { title, template, url_suffix: urlSuffix } = JSON.parse(configurationString ?? '{}') as NOTIFIER_CONNECTOR_EMAIL_INTERFACE;
 
-  const renderedTitle = ejs.render(title, templateData);
-  const octiTool = new NotificationTool();
-  const renderedEmail = ejs.render(template, { ...templateData, url_suffix: urlSuffix, octi: octiTool });
+  const sanitizedData = {
+    // Sanitize template data before rendering
+    ...sanitizeNotificationData(templateData),
+    url_suffix: urlSuffix,
+    octi: new NotificationTool(),
+  };
+
+  let renderedTitle: string;
+  let renderedEmail: string;
+
+  try {
+    renderedTitle = await safeRender(title, sanitizedData);
+    renderedEmail = await safeRender(template, sanitizedData);
+  } catch (error) {
+    logApp.error('[OPENCTI-MODULE] Publisher manager email rendering error', {
```

Comment:
do we need to catch the error here ?
why use error level ?
(idem on other calls that follow the same pattern, note that is not always done, eg in user.js)